src/commonMain/kotlin/dev/romainguy/kotlin/math/Quaternion.kt

```diff
@@ -287,56 +328,51 @@ fun cross(a: Quaternion, b: Quaternion): Quaternion {
  * @param a The beginning value
  * @param b The ending value
  * @param t The ratio between the two floats
- * @param valueEps Prevent blowing up when slerping between two quaternions that are very near each
- * other. Linear interpolation (lerp) is returned in this case.
+ * @param dotThreshold If the quaternion dot product is greater than this value
+ * (i.e. the quaternions are very close to each other), then the quaternions are
+ * linearly interpolated instead of spherically interpolated.
  *
  * @return Interpolated value between the two floats
  */
-fun slerp(a: Quaternion, b: Quaternion, t: Float, valueEps: Float = 0.0000000001f): Quaternion {
+fun slerp(a: Quaternion, b: Quaternion, t: Float, dotThreshold: Float = 0.9995f): Quaternion {
```

Comment:
This is an API breakage btw

Reply:
Yes. I know. I'm very sorry about it but dot threshold is really more commonly used and I had rarely some wrong behaviors with the previous implementation when testing AR placement.

Reply:
The dot threshold is commonly `0.9995` and is usually defined as a constant so I can remove the parameter to remove the API breakage.

Reply:
No that's fine, but we should be careful going forward

Reply:
Sure